inc/customizer/options/js/2_layout.js

```diff
@@ -89,6 +89,12 @@ var layoutRanges = {
 	},
 };
 
+
```

Comment:
@cristian-ungureanu 

This is how I would abstract the live preview part for this feature. I think it's more organized and less computing-intensive. It's also more in line with how the other live preview bits were thought out in this file. 💃 

https://gist.github.com/abaicus/8110bcea97e798bf75885ce545700040#file-live-preview-js-L144-L182